src/webgpu/api/operation/buffers/map_oom.spec.ts

```diff
@@ -38,7 +38,7 @@ g.test('mapAsync')
     const { oom, write, size, unmapBeforeResolve } = t.params;
 
     const buffer = t.expectGPUError(
-      'out-of-memory',
+      'validation',
```

Comment:
Why? This seems like an out-of-memory error to me (trying to allocate a buffer of size ~8PiB or 128GiB)

Reply:
According to spec: https://www.w3.org/TR/webgpu/#abstract-opdef-validating-gpubufferdescriptor both of these are greater than maxBufferSize so it should actually be a validation error, not an oom error.

Reply:
Oh, right, of course. This test was written before we had that limit. This change breaks the intent of the test though :( We should at least update the test description.

I guess we have no way to reliably hit an OOM, but we should add a TODO to have a test that enables the highest available maxBufferSize limit and at least _tries_ to get an OOM.

Reply:
Finally addressed the concerns here! Ultimately removed the old test entirely and moved it into stress. I tried to run the stress test and it occasionally passes? Other times it crashes the GPU process... The test itself should be ok though so let me know what you think.